scm/src/scm_forcing.F90

```diff
@@ -324,7 +326,7 @@ subroutine interpolate_forcing(scm_input, scm_state, in_spinup)
           end do
         end if
 
-        if (scm_state%surface_thermo_control == 0 .or. scm_state%surface_thermo_control == 1 .or. scm_state%surface_thermo_control == 2) then
+        if (scm_state%surface_thermo_control == 0 .or. scm_state%surface_thermo_control == 1 .or. scm_state%surface_thermo_control == 2 .and. .not. scm_state%do_sst_initialize_only) then
```

Comment:
Because of operator precedence this is an equivalent statement to `if (scm_state%surface_thermo_control == 0 .or. scm_state%surface_thermo_control == 1 .or. (scm_state%surface_thermo_control == 2 .and. .not. scm_state%do_sst_initialize_only)) then`

Adding the parenthesis around the `.and.` statement might help make it clearer to people quickly glancing at the code?